hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -163,6 +163,19 @@
   private final RegionStates regionStates = new RegionStates();
   private final RegionStateStore regionStateStore;
 
+  /**
+   * When the operator uses this configuration option, any version between
+   * the current version and the new value of
+   * "hbase.min.version.move.system.tables" does not trigger any region movement.
```

Comment:
Do you want to explain what the 'region movement' is here? The auto-migration of system tables to newer server versions. A dev reading this w/o context might not tie the 'reigon movement' to the auto-movement to newer versions.